contracts/implementations/nodes/AftermarketDevice.sol

```diff
@@ -246,6 +246,86 @@ contract AftermarketDevice is
         emit AftermarketDeviceClaimed(aftermarketDeviceNode, owner);
     }
 
+    /// @notice Pairs an aftermarket device with a vehicle through a metatransaction.
+    /// Both vehicle and AD owners can pair.
```

Comment:
The AD owner can't pair, can he? Unless he happens to own the vehicle NFT too.

Reply:
Oops... comment in the wrong place. Moved to unpair.